bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/DoublyChargedHiggsGMmodel_HWW/Card_gen.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+
+for mass in 200 300 400 500 600 700 800 900 1000 1500 2000 3000
+do
+    echo $mass
+    mkdir -p DoublyChargedHiggsGMmodel_HWW_M${mass}
+    
```

Comment:
I suggest you make arrays for mass, width and couplings and then one simple for-loop in which you replace 
MASSBLBL with MASS[i]
WIDTHBLBL with WIDTH[i]. 
Your implementation is very error-prone.